lib/resources/example_json/r4_examples_encounter.rb

```diff
@@ -943,6 +954,17 @@ module Resources
                 }
               }
             ],
+            "appointment": [
+              {
+                "reference": "Appointment/16267757"
+              },
+              {
+                "reference": "Appointment/16267758"
+              },
+              {
+                "reference": "Appointment/16267759"
+              }
+            ],
```

Comment:
I think it will be okay if we change it in once place. Lets just keep the read example change,
https://github.com/cerner/fhir.cerner.com/blob/a719a0ba5d0d34734a48089e5ff54c84b01d5471/lib/resources/example_json/r4_examples_encounter.rb#L303(as it is used by search also https://github.com/cerner/fhir.cerner.com/blob/a719a0ba5d0d34734a48089e5ff54c84b01d5471/lib/resources/example_json/r4_examples_encounter.rb#L364) and remove the other one

Reply:
fix for this 

bc0c58d[](https://github.com/cerner/fhir.cerner.com/pull/467/commits/bc0c58d0e3572ffb892fcda5cc0a1e717fcb5fdb)

Reply:
Can you update the screenshots?

Reply:
<img width="1180" alt="Screen Shot 2020-09-23 at 9 32 12 AM" src="https://user-images.githubusercontent.com/71471042/94019614-fc130400-fd7f-11ea-9a14-22db0e528e38.png">
<img width="1180" alt="Screen Shot 2020-09-23 at 9 33 18 AM" src="https://user-images.githubusercontent.com/71471042/94019623-fe755e00-fd7f-11ea-9f3b-44564e073134.png">